projects/ngrx.io/content/guide/component/index.md

```diff
@@ -1,19 +1,38 @@
 # @ngrx/component
 
-Component is a set of primitive reactive helpers to enable fully reactive, fully zone-less applications. They give more control over rendering, and provide further reactivity for Angular applications.
+Component is a set of primitive reactive helpers to enable fully reactive,  applications. They give more control over rendering, and provide further reactivity for Angular applications.
```

Comment:
extra space here, but I think we should live "Zoneless" in there. That's the main point goal for both `ngrxLet` and `ngrxPush`.